src/job_cache/schema.sql

```diff
@@ -20,8 +20,9 @@ create table if not exists jobs(
   commandline  blob    not null,
   environment  blob    not null,
   stdin        text    not null,
-  bloom_filter integer);
-create index if not exists job on jobs(directory, commandline, environment, stdin);
+  bloom_filter integer not null,
+  runner_hash  text    not null);
+create index if not exists job on jobs(directory, commandline, environment, stdin, runner_hash);
```

Comment:
discussed offline but we should try to do a migration here instead so folks don't loose their current caches

Something like:
1) add column if not exists
2) drop index if exists
3) add index if not exists

Reply:
As I went to go fix this, it turns out that sqlite3 does not have the required operations to do this. We will just give people a one-time command to run if they want to migrate. Eventually we'll have global caching and we can migrate that via the proper mechanisms and we can repopulate nuked local caches from that.